app/src/main/res/values/strings.xml

```diff
@@ -443,6 +443,10 @@
     <string name="activity_backup_drive_button_open_drive">Manage on Drive</string>
     <string name="mmol_mol">mmol/mol</string>
     <string name="mmol_L">mmol/L</string>
+    <string name="mmol_mol_value">%1$s mmol/mol</string>
```

Comment:
Ah no, looks like we operate with strings everywhere. But these number strings are they localised?